packages/components/src/alignment-matrix-control/styles/alignment-matrix-control-styles.ts

```diff
@@ -7,15 +7,15 @@ import { css } from '@emotion/react';
 /**
  * Internal dependencies
  */
-import { COLORS } from '../../utils';
+import { COLORS, CONFIG } from '../../utils';
 import type {
 	AlignmentMatrixControlProps,
 	AlignmentMatrixControlCellProps,
 } from '../types';
 
 export const rootBase = () => {
 	return css`
-		border-radius: 2px;
+		border-radius: ${ CONFIG.radiusMedium };
```

Comment:
Did you mean to increase this to `4px`? I don't see it in the list of intended changes.

Reply:
My feeling is this falls into 'small container' territory. That said, I don't think the radius of this component is visually rendered anywhere?

Reply:
I agree, but it seems safer to adjust the value here rather than remove it? We could remove it in a follow up with a dedicated changelog entry?

Reply:
I don't mind keeping it if that's what you prefer 👍